_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,238 @@
+---
+name: iframe has a unique accessible name
```

Comment:
Needs to be updated to match updates to rule. 

Suggestion: 

> Iframes with identical accessible names serve equivalent purpose

(aligned with https://github.com/auto-wcag/auto-wcag/pull/220/files, that however misses some reviews)